src/app/App.scss

```
@@ -1,5 +1,16 @@
 @import "/src/styles/index";
+@import '~antd/dist/antd.css';
 
-h1 {
-    @include flex-center;
+.ant-btn-primary{
```

```suggestion
.ant-btn-primary {
```
Наверное, приятнее читать с отступом